converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -0,0 +1,166 @@
+	ACI_SECTION_MISSING_REPORTING_PARAMETER_ACT("The ACI Section must have one Reporting Parameter ACT"),
+	CLINICAL_DOCUMENT_MISSING_ACI_OR_IA_OR_ECQM_CHILD("Clinical Document Node must have at least one Aci or IA or eCQM Section Node as a child"),
+	CLINICAL_DOCUMENT_MISSING_PROGRAM_NAME("Clinical Document must have one and only one program name"),
+	CLINICAL_DOCUMENT_INCORRECT_PROGRAM_NAME("Clinical Document program name is not recognized"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ACI_SECTIONS("Clinical Document contains duplicate ACI sections"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_IA_SECTIONS("Clinical Document contains duplicate IA sections"),
+	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS("Clinical Document contains duplicate eCQN sections"),
+	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_START("Must have one and only one performance start"),
+	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_END("Must have one and only one performance end"),
+	REPORTING_PARAMETERS_MISSING_PERFORMANCE_YEAR("Must have a performance year"),
+	QUALITY_MEASURE_SECTION_REQUIRED_REPORTING_PARAM_REQUIREMENT("The Quality Measure Section must have only one Reporting Parameter ACT"),
+	PERFORMANCE_RATE_INVALID_VALUE("Must enter a valid Performance Rate value"),
+	CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS("Must contain a practice site address for CPC+ conversions"),
+	CPC_CLINICAL_DOCUMENT_ONLY_ONE_APM_ALLOWED("One and only one Alternative Payment Model (APM) Entity Identifier should be specified"),
+	CPC_CLINICAL_DOCUMENT_ONE_MEASURE_SECTION_REQUIRED("Must contain one Measure (eCQM) section"),
+	CPC_QUALITY_MEASURE_ID_INVALID_PERFORMANCE_RATE_COUNT("Must contain correct number of performance rate(s). Correct Number is %s", true),
+	NUMERATOR_DENOMINATOR_MISSING_CHILDREN("This %s Node does not have any child Nodes", true),
+	NUMERATOR_DENOMINATOR_INCORRECT_CHILD("This %s Node does not have an Aggregate Count Node", true),
+	NUMERATOR_DENOMINATOR_TOO_MANY_CHILDREN("This %s Node has too many child Nodes", true),
+	NUMERATOR_DENOMINATOR_MUST_BE_INTEGER("This %s Node Aggregate Value is not an integer", true),
+	NUMERATOR_DENOMINATOR_INVALID_VALUE("This %s Node Aggregate Value has an invalid value", true),
+	IA_SECTION_MISSING_IA_MEASURE("The IA Section must have at least one IA Measure"),
+	IA_SECTION_MISSING_REPORTING_PARAM("The IA Section must have one Reporting Parameter ACT"),
+	IA_SECTION_WRONG_CHILD("The IA Section must contain only measures and reporting parameter"),
+	NPI_TIN_COMBINATION_MISSING_CLINICAL_DOCUMENT("Clinical Document Node is required"),
+	NPI_TIN_COMBINATION_EXACTLY_ONE_DOCUMENT_ALLOWED("Only one Clinical Document Node is allowed"),
+	CPC_QUALITY_MEASURE_ID_MISSING_STRATA("Missing strata %s for %s measure (%s)", true),
+	CPC_QUALITY_MEASURE_ID_STRATA_MISMATCH("Amount of stratifications %d does not meet expectations %d for %s measure (%s). Expected strata: %s", true),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 149 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)